libs/domains/observability/feature/src/lib/service-overview/local-chart/local-chart.tsx

```diff
@@ -3,11 +3,11 @@ import clsx from 'clsx'
 import { OrganizationEventTargetType } from 'qovery-typescript-axios'
 import { type PropsWithChildren, useState } from 'react'
 import { useParams } from 'react-router-dom'
-import { CartesianGrid, ComposedChart, ReferenceLine, XAxis, YAxis } from 'recharts'
+import { CartesianGrid, ComposedChart, ReferenceArea, ReferenceLine, XAxis, YAxis } from 'recharts'
 import { type AnyService } from '@qovery/domains/services/data-access'
 import { useService } from '@qovery/domains/services/feature'
-import { Button, Chart, Heading, Icon, Section, Tooltip } from '@qovery/shared/ui'
-import { createXAxisConfig } from '@qovery/shared/ui'
+import { Button, Chart, Heading, Icon, Section, Tooltip, useZoomableChart } from '@qovery/shared/ui'
+import { createXAxisConfig, getTimeGranularity } from '@qovery/shared/ui'
```

Comment:
You've two imports of `@qovery/shared/ui` merge createXAxisConfig, getTimeGranularity with the import below!